src/components/Persona/Persona.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PerrsonLink/PersonLink';
+
+type Props = {
+  person: Person;
+};
+
+export const Persona: React.FC<Props> = React.memo(
+  ({ person }) => {
+    const { slug = '' } = useParams();
+    const isSelected = (person.slug === slug);
+
+    return (
+      <tr
+        data-cy="person"
+        className={classNames({
+          'has-background-warning': isSelected,
+        })}
+      >
+        <td>
+          <PersonLink person={person} />
+        </td>
+
+        <td>{person.sex}</td>
```

Comment:
Would be better to destructure the person

Reply:
Did it in the PeopleTable